src/SUMMARY.md

```diff
@@ -25,6 +25,12 @@
     * [Accessing Glean data](concepts/glean/accessing_glean_data.md)
     * [Glean Debug ping viewer](concepts/glean/debug_ping_view.md)
   * [Growth & Usage Dashboard (GUD)](tools/gud.md)
+* [Metrics](metrics/index.md)
```

Comment:
My gut read is to have the following format:

* Metrics
  * Metric 1
  * Metric 2
  * ...
  * Common Slices (Usage Critera and Slicing Dimensions)

I'm not sure if Metrics Standardization and Policy belongs in DTMO. What does that section cover? If it's "how to engage or change these metrics", then let's add it to `metrics/index.md`

Reply:
Sounds fair.  I might push for separating usage criteria from slicing dimensions though.  I feel that the definitions around when we consider a user to have used our products may be important - especially if we want to move beyond "had the app/browser open", which I think we should do.

For the how to engage or change these metrics, that seems like a much more unusual and specialized task, so wanted a seperate section for it.  Why do you feel it should be in the index.md?

Reply:
> For the how to engage or change these metrics, that seems like a much more unusual and specialized task, so wanted a seperate section for it. Why do you feel it should be in the index.md?

Good point. I think the specialization suggests it may not be a good fit for DTMO, maybe we should maintain the docs for changing metrics elsewhere and link from `index.md`

>  I might push for separating usage criteria from slicing dimensions

Could be. Usage criteria feels like an instance of a slicing dimension to me. E.g. I don't see much difference between slicing to "users who saw at least 5 URIs" and "users who used the browser in DE" but I'm open to being convinced otherwise. 

For now, I'd prefer to add as few pages as possible to keep cognitive load low.

Reply:
> I don't see much difference between slicing to "users who saw at least 5 URIs" and "users who used the browser in DE" but I'm open to being convinced otherwise.

In my mental model (which is different to GUD's), a metric is "the thing you want to measure" and sometimes you'll want to _add_ a slicing dimension to measure that thing over a subset of the user population.

If we're investigating a change in a metric, we'll want to slice that metric over many dimensions (check that metric just for German users, check it just for users in the regular users v3 segment, etc) to try to find a slice that isolates the change in the metric.

In the GUD model, which is basically what is presented here, a "usage criterion" is part of the metric's definition - swapping out a usage criterion changes the metric itself, rather than splitting the population into two.

Note that this "try lots of slices to find one that isolates the change" requires that the slices to be defined carefully to avoid tautology ("why yes sir, 100% of the increase in people sending pings today was from people who sent pings today"). If they're defined based on behaviour rather than enduring properties (e.g. country) then they should be defined on _past behaviour_, like I've tried to do with the segments work.

Reply:
Regarding having the policy docs elsewhere, I'm not really opposed, but feel that it's nice to have an identified home for MSG and this seems like it will be it.  It seems natural that people might expect to find documents related to our policies here - a nice parallel is the docs on contributing to DTMO that are linked at the bottom.  But I don't feel that strongly.  Any particular reason not to have the policy docs here?

Regarding usage criteria, the reason they're important is to be explicit that we are counting something in particular.  When we use MAU right now, we are implicitly saying that we are counting all desktop users that submitted a ping.  Implicit definitions are not a good thing.  The usage critera forces us to be explicit about what we are counting.

And even if we weren't using the GUD data model, we really need a list of usage criteria as a first-class MSG citizen.  One of the problems we explicitly want to tackle is understand what we should count as an "active user".   This flared up recently with glean.  Even if we allow "any ping" to be the standard usage criteria for a product, it should be explicit.

I absolutely agree that this space should properly support non-GUD metrics.  I'll try to make clearer that the usage criteria section is not necessarily a component of every metric.

Reply:
Will note for @felixlawrence that I have dramatically changed definitions.md to make clear that usage criteria are not a fundamental part of the general metric definition.

Reply:
> Regarding having the policy docs elsewhere, I'm not really opposed, but feel that it's nice to have an identified home for MSG and this seems like it will be it. It seems natural that people might expect to find documents related to our policies here - a nice parallel is the docs on contributing to DTMO that are linked at the bottom. But I don't feel that strongly. Any particular reason not to have the policy docs here?

I do worry a little bit about diluting docs.telemetry.mozilla.org's purpose / focus, as has happened in the past. But I can see the argument that this is the simplest place to put them for now.

Reply:
Adding some late comments for the record. These aren't pressing; no need to address these points.

> Regarding having the policy docs elsewhere, I'm not really opposed, but feel that it's nice to have an identified home for MSG and this seems like it will be it. 

I agree that we should have a home for MSG, but I don't think it should be DTMO. I share @wlach's concern about diluting DTMO. 

>  a nice parallel is the docs on contributing to DTMO that are linked at the bottom.

Good point. The `meta` docs are probably too extensive in DTMO. We should probably prefer to cover the contributing guidelines in the repository instead of in the rendered documentation.